content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -464,6 +464,30 @@ go run ./write.go
 
 <!------------------------------- END GO CONTENT ------------------------------>
 {{% /tab-content %}}
+{{% tab-content %}}
+<!------------------------------- BEGIN TELEGRAF CONTENT ------------------------------>
+[Telegraf](/{{< latest "telegraf" >}}/) with the [`file` input plugin](/{{< latest "telegraf" >}}/plugins/#input-file) consume line protocol or CSV, and then use the `influxdb_v2` output plugin to write the data to InfluxDB.
+
+The following example shows a minimal [`outputs.influxdb_v2`](/{{< latest "telegraf" >}}/plugins/#output-influxdb_v2) configuration for writing data to your InfluxDB Cloud Dedicated cluster:
+
+```toml
+[[outputs.influxdb_v2]]
+  urls = ["https://cluster-id.influxdb.io"]
+  token = "${INFLUX_TOKEN}"
+  organization = ""
+  bucket = "DATABASE_NAME"
+```
+
+Replace the following:
+
+- **`DATABASE_NAME`**: your InfluxDB Cloud Dedicated [database](/influxdb/cloud-dedicated/admin/databases/)
+
+In the example, **`INFLUX_TOKEN`** is an environment variable assigned to an InfluxDB API token that has _write_ permission to the bucket.
+
+The Telegraf `file` input plugin and the `output-influxdb_v2` output plugin provide many options for reading and writing data.
+To learn more, see how to [use Telegraf to write CSV data](/influxdb/cloud-dedicated/write-data/csv/telegraf/).
```

Comment:
The Get started documentation should be a step by step process that users can follow and be successful. I think to make this fool-proof, we need to walk through each step a user needs to go through:

1. If you haven't already, download and install Telegraf.
2. Copy and save the line protocol above to a file on your local system.
3. Create a new Telegraf configuration file with just the `inputs.file` and `outputs.influxdb_v2` plugins. (We should provide the contents of the file with a placeholder path to the line protocol file.
4. Execute the `telegraf` daemon with the `--once` flag  to run a single Telegraf collection cycle and write the line protocol to {{< cloud-name >}}:

    ```sh
    telegraf --once
    ```